library/nsx_ospf.py

```diff
@@ -69,6 +69,15 @@ def check_router_id(current_config, router_id):
         current_config['routing']['routingGlobalConfig']['routerId'] = router_id
         return True, current_config
 
+def check_ecmp(current_config, ecmp):
```

Comment:
Very Minor NIT; Can you check PEP8 compliance? In this case PEP8 is complaining that there is only one blank line before the new function